awscli/examples/greengrass/create-device-definition-version.rst

```diff
@@ -0,0 +1,79 @@
+**To create a device definition version**
+
+The following ``create-device-definition-version`` example creates a device definition version and associates it with the specified device definition. The version defines two devices. 
+Before you can create a Greengrass device, you must first create and provision the corresponding AWS IoT thing. This process includes the following ``iot`` commands that you must run to get the required information for the Greengrass command:
+ 
+* Create the AWS IoT thing that corresponds to the device::
+
+    aws iot create-thing \
+        --thing-name "InteriorTherm"
+
+Output::
+
+   {
+       "thingArn": "arn:aws:iot:us-west-2:123456789012:thing/InteriorTherm",
+       "thingName": "InteriorTherm",
+       "thingId": "01d4763c-78a6-46c6-92be-7add080394bf"
+   }
+    
+* Create public and private keys and the device certificate for the thing. This example uses the `create-keys-and-certificate` command and requires write permissions to the current directory. Alternatively, you can use the `create-certificate-from-csr` command::
+
+    aws iot create-keys-and-certificate \
+        --set-as-active \
+        --certificate-pem-outfile "myDevice.cert.pem" \
+        --public-key-outfile "myDevice.public.key" \
+        --private-key-outfile "myDevice.private.key"
+
+Output::
+
+    {
+        "certificateArn": "arn:aws:iot:us-west-2:123456789012:cert/66a415ec415668c2349a76170b64ac0878231c1e21ec83c10e92a18bd568eb92",
+        "certificatePem": "-----BEGIN CERTIFICATE-----\nMIIDWTCAkGgAwIBATgIUCgq6EGqou6zFqWgIZRndgQEFW+gwDQYJKoZIhvc...KdGewQS\n-----END CERTIFICATE-----\n",
+        "keyPair": {
+            "PublicKey": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBzrqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAqKpRgnn6yq26U3y...wIDAQAB\n-----END PUBLIC KEY-----\n",
+            "PrivateKey": "-----BEGIN RSA PRIVATE KEY-----\nMIIEowIABAKCAQEAqKpRgnn6yq26U3yt5YFZquyukfRjbMXDcNOK4rMCxDR...fvY4+te\n-----END RSA PRIVATE KEY-----\n"
+        },
+        "certificateId": "66a415ec415668c2349a76170b64ac0878231c1e21ec83c10e92a18bd568eb92"
+    }
+
+* Create an AWS IoT policy that allows `iot` and `greengrass` actions. For simplicity, the following policy allows actions on all resources, but your policy can be more restrictive::
```

Comment:
This single backtick here looks like markdown style. Can we add double backticks? It looks like this is done in multiple of these pages.

Reply:
Ok.  I think I found and fixed all of those.  Thanks for the catch!